source/SIL.AppBuilder.Portal.Frontend/src/data/models/workflow-definition.ts

```diff
@@ -1,13 +1,15 @@
-import { AttributesObject, ResourceObject } from 'jsonapi-typescript';
+import { AttributesObject, ResourceObject } from "jsonapi-typescript";
```

Comment:
dont use " better use '

Reply:
the linter should catch this. No need to be a human linter :)